ccloud/docs/beginner-cloud.rst

```diff
@@ -189,7 +189,7 @@ Create a new API key/secret pair for user and specify it as the default
          "secret": "fjcDDyr0Nm84zZr77ku/AQqCKQOOmb35Ql68HQnb60VuU+xLKiu/n2UNQ0WYXp/D"
       }
 
-#. Associate the API key ``QX7X4VA4DFJTTOIA`` to the |ak| cluster ``lkc-x6m01``:
+#. Specify that you want to use the API key ``QX7X4VA4DFJTTOIA`` for the |ak| cluster ``lkc-x6m01``:
```

Comment:
#. Specify you want to use the API key ``QX7X4VA4DFJTTOIA`` for the |ak|
   cluster ``lkc-x6m01``: